manifests/d/dwgltd/sgfg/6.0.0/dwgltd.sgfg.locale.en-US.yaml

```diff
@@ -0,0 +1,17 @@
+PackageIdentifier: "dwgltd.sgfg"
+PackageVersion: "6.0.0"
+PackageLocale: "en-US"
+Publisher: "Dawugui Ltd"
+PublisherUrl: "https://www.transfer.taipei/"
```

Comment:
```suggestion
# PublisherUrl: "https://www.transfer.taipei/"
```

Reply:
This should fix the error from cloudflare

Reply:
THX.